cel/testdata/testprotovalue.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto2";
```

Comment:
Instead of introducing a new protobuf, consider re-using `test/proto2pb/test_all_types.pb.go`. Doing so will change your test expression slightly, but hopefully not significantly..

Reply:
Done.